src/js/App.js

```diff
@@ -50,18 +51,61 @@ window.App = Backbone.View.extend({
     },
 
     fadeInListView: function() {
+        this.regionListView = new RegionListView({ model: this.model });
+        this.regionListView.render();
         this.regionListView.$el.fadeIn(200);
     },
 
+    animateDetailsViewReplacingList: function() {
+        var self = this;
+        var selectedRegion = this.model.get('selectedRegion').get('title');
+        var listItems = $('#region-location-list').children();
+        var $sidebarInnerEl = $('.sidebar--inner');
```

Comment:
If possible, consider replacing these with calls like `this.$el.find()`. It should be marginally faster.

Reply:
Tried this out and it wasn't able to find the list items to loop through afterward, so it just made the update without the animation. Going to leave as is for now but we can revisit if we encounter performance issues.